### PR TITLE
Navbar link's scroll position resolved.

### PR DESCRIPTION
**Pull request by Ashwinib26**
To resolve the issue of incorrect scroll position when navigating between pages, I used React's useEffect hook to automatically scroll the window to the top whenever a new page loads, ensuring correct view positioning in all the components pages.
Now, it is working as desired.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `.card` CSS class for smooth transition effects and hover shadow.
	- Updated multiple components to scroll to the top on mount for better navigation.
	- Enhanced `TodaysSpecial` component with pricing information and improved hover effects.

- **Bug Fixes**
	- Fixed rendering issues related to hover states in the `TodaysSpecial` component.

- **Documentation**
	- Updated import statements to include the `useEffect` hook where necessary.

### CODE REVIEW

**Comment by Ashwinib26** [comment]
@RamakrushnaBiswal  , I have resolved the conflicts.

File: frontend/src/components/Pages/Login.jsx [hunk 1]
```diff
@@ -1,8 +1,14 @@
+
+import React, { useState , useEffect } from "react";
+import { Link } from "react-router-dom";
+import photo from "../../assets/login.png";
```
Comment (RamakrushnaBiswal) [inline frontend/src/components/Pages/Login.jsx:4]: @Ashwinib26 you add twice it remove and follow code rabit suggestions

**Comment by RamakrushnaBiswal** [comment]
@Ashwinib26 also resolve the conflicts :)

**Comment by Ashwinib26** [comment]
@RamakrushnaBiswal  , I have resolved all the conflicts.

**Pull request closed without merging**